client/my-sites/upgrades/checkout/free-trial-confirmation-box.js

```diff
@@ -40,8 +42,14 @@ var FreeTrialConfirmationBox = React.createClass( {
 		);
 	},
 
-	render: function() {
-		var classSet = classNames( {
+	getProductName() {
+		const planProduct = find( this.props.cart.products, isPlan );
+
+		return ( planProduct && planProduct.product_name ) || '';
```

Comment:
As far I could see this is the case only when `this.props.selected` is set to `false`;